### PR TITLE
OU-516: Link to Gantt Chart when clicking on a bubble in the ScatterPlot chart

### PR DESCRIPTION
**Pull request by andreasgerstmayr**
[OU-516](https://issues.redhat.com//browse/OU-516): Link to Gantt Chart when clicking on a bubble in the ScatterPlot chart

Resolves: https://issues.redhat.com/browse/OU-516

### CODE REVIEW

File: web/src/pages/TraceDetailPage.tsx [hunk 1]
```diff
@@ -25,6 +25,11 @@ interface RouteParams {
   traceId: string;
 }
 
+export function LinkToTraceDetailPage(traceId: string) {
```
Comment (jgbernalp) [inline web/src/pages/TraceDetailPage.tsx:28]: nit
```suggestion
export function linkToTraceDetailPage(traceId: string) {
```

File: web/src/pages/TracesPage/TraceTable.tsx [hunk 2]
```diff
@@ -47,5 +48,5 @@ export function TraceTable({ runQuery }: TraceTableProps) {
 }
 
 export function traceDetailLink({ traceId }: { traceId: string }) {
```
Comment (jgbernalp) [inline web/src/pages/TracesPage/TraceTable.tsx:50]: we might not need this function as is just wrapping `linkToTraceDetailPage`, should we adjust the signature?

Comment (andreasgerstmayr) [reply web/src/pages/TracesPage/TraceTable.tsx:50]: The signatures of `traceLink` of `<PersesTraceTable>` and `onClick` of `<ScatterChart>` are different, that's why I thought it's a good idea to create tiny wrapper functions to align them with an internal function, which only takes one required argument, the `traceId`.

But I'm not sure if `LinkToTraceDetailPage` should be in `TraceDetailPage.tsx` or it'd be better to put it in `TracingUI.tsx` or in a new file like `links.ts` or something.

File: web/src/pages/TraceDetailPage.tsx (hunk 1 above)
Comment (andreasgerstmayr) [reply web/src/pages/TraceDetailPage.tsx]: done

File: web/src/pages/TracesPage/TraceTable.tsx (hunk 2 above)
Comment (jgbernalp) [reply web/src/pages/TracesPage/TraceTable.tsx:50]: maybe a links file would be better I see this changing in the future to consider dev console.

Comment (andreasgerstmayr) [reply web/src/pages/TracesPage/TraceTable.tsx:50]: done

**Pull request merged**